model/lifecycle.go

```diff
@@ -338,13 +338,30 @@ func RestartVersion(versionId string, taskIds []string, abortInProgress bool, ca
 			return errors.WithStack(err)
 		}
 	}
+	allFinishedTasks, err := getTasksToReset(taskIds)
+	if err != nil {
+		return errors.Wrap(err, "getting finished tasks")
+	}
+	if len(allFinishedTasks) == 0 {
+		return nil
+	}
+	err = restartTasks(allFinishedTasks, caller)
+	if err != nil {
+		return errors.Wrap(err, "restarting tasks")
+	}
+	return errors.Wrap(setVersionStatus(versionId, evergreen.VersionStarted), "changing version status")
```

Comment:
TODO: don't we need to call updateBuildAndVersionStatus

Reply:
Not sure what you mean here?

Reply:
I think this was a thinking comment i forgot to make intelligible and idk what it meant now haha